nativescript-angular/element-registry.ts

```diff
@@ -153,7 +153,9 @@ export function getSingleViewRecursive(nodes: Array<any>, nestLevel: number): Vi
 
     const parentLayout = rootLayout.parent;
     if (parentLayout instanceof LayoutBase) {
+        let node = rootLayout.parentNode;
         parentLayout.removeChild(rootLayout);
+        rootLayout.parentNode = node;
```

Comment:
Hi @NathanaelA ,

The reason for this fix is the following:
Setup:
- In components like RadListView/ListView the component itself is going to handle the adding of the created templates to the UI tree
- In those cases it is necessary to "remove" the view that is created by the Angular's createEmbeddedView because by design it automatically adds the created element to the UI tree
- This is done via the `getSingleViewRecursive`

So here is the source of this leak:
- The created "queue" of parent > child relationship that is created inside `nativescript-angular` initially (before getSingleViewRecursive removes the view) sets the parentNode to the actual ContainerViewRef that is calling the `createEmbeddedView`
- Due to the above reason we have to remove that view from the UI tree which itself resets the parentNode to undefined because the elements (the template View) is removed from its parent
- Later on when the GC is triggered that View does not have its correct "parentNode" (the one that was set when the creation of `createEmbeddedView` was called) and it cannot be correctly removed from the "queue". 

So this fix makes sure that even if we "remove a view" from the UI by using the `getSingleViewRecursive` its "representation" in the internal elements queue will be correct and will be GC collectable.